aries-site/src/components/content/BodyText.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Paragraph } from 'grommet';
+
+// Body text size should be based on if its parent heading is an
+// h2, h3, etc.
+const TEXT_SIZE = {
+  1: 'large',
+  2: 'medium', // default font size
+  3: 'small',
+};
+
+export const BodyText = ({ children, level, size, ...rest }) => {
+  return (
+    <Paragraph size={size || TEXT_SIZE[level]} fill margin="none" {...rest}>
+      {children}
+    </Paragraph>
+  );
+};
+
+BodyText.propTypes = {
+  children: PropTypes.string.isRequired,
```

Comment:
could be removed as well.

Reply:
Regardless, I don't think it should be limited to a string. (it could be a react Node).

Reply:
Or was it on purpose because the child will go to Paragraph?

Reply:
Yes it was on purpose that the child will go to Paragraph. The BodyText component would be for any text blocks within a subsection. The idea behind creating a specialized component for it is that when BodyText is placed within a Subsection, the level on the Subsection will determine what size text to use. So, if a BodyText is used within a Subsection with level 2, then the paragraph size will be the text size meant to be paired with h2.

Reply:
Based on Chris's designs, what goes inside of BodyText would be a block of text.